docs/source/index.rst

```diff
@@ -21,6 +21,7 @@ Contents
    interactive/index
    config/index
    development/index
+   development/release_process
```

Comment:
I don't think this needs a separate entry in the top-level TOC - let's add it to `development/index` instead.